docs/entities/controls.rst

```diff
@@ -42,6 +42,19 @@ A series of buttons are available by default. They are:
     - Power On/Off the printer.
     - From Moonraker API (*machine.device_power.devices*)
 
+Macros
+---------------------------------
+
+Each macro configured in Klipper is available as a button.
+
+Depending of the macro, it may be disabled by default. If Macro Description is `G-Code macro` it will be enabled by default. It will be disabled otherwise.
```

Comment:
If the Macro description is `G-Code macro` it will be enabled by default, otherwise it will be disabled.